### PR TITLE
Translate 2.6.0-preview3 post (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_cn/news/_posts/2018-11-06-ruby-2-6-0-preview3-released.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  此方法以一个二元组数组 `__FILE__` 和 `__LINE__` 的形式返回 binding 的源代码路径。传统上，这可以通过执行 `eval("[__FILE__, __LINE__]", binding)` 来获得相同的数据。但我们计划改变这一行为让 `Kernel#eval` 忽略 `binding` 的源代码路径 [[漏洞 #4352]](https://bugs.ruby-lang.org/issues/4352)。所以，用户需要通过新加入的方法来替代之前的 `Kernel#eval`。
+
+* 增加 `:exception` 选项，以让 `Kernel.#system` 抛出错误而不是返回 `false`。[[功能 #14386]](https://bugs.ruby-lang.org/issues/14386)
+
+## 性能提升
+
+* 提升 `Proc#call` 的速度，因为我们无需再关心 `$SAFE`。[[功能 #14318]](https://bugs.ruby-lang.org/issues/14318)
+
+  通过 `lc_fizzbuzz` 多次使用 `Proc#call` 的基准测试我们测量到了 1.4 倍性能提升 [[漏洞 #10212]]((https://bugs.ruby-lang.org/issues/10212))。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2018-11-06-ruby-2-6-0-preview3-released.md]: Trivial, just notice you interchangeably use *benchmark* and *基准测试*, maybe just one?

File: zh_cn/news/_posts/2018-11-06-ruby-2-6-0-preview3-released.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+* 提升了当 `block` 是代码块参数时 `block.call` 的性能。[[功能 #14330]]((https://bugs.ruby-lang.org/issues/14330))
+
+  Ruby 2.5 提升了代码块传递的性能。[[功能 #14045]](https://bugs.ruby-lang.org/issues/14045) 另外，Ruby 2.6 提升了传递代码块调用时的性能。通过 micro-benchmark 我们观察到了 2.6 倍性能提升。
+
+* 引入了瞬态堆 (theap)。 [Bug #14858] [Feature #14989] 瞬态堆是用于管理指向特定类（Array, Hash, Object 和 Struct）短生命周期内存对象的堆。例如，创建小而短生命周期的哈希对象的速度提升到了 2x 快。根据 rdoc benchmark，我们观察到了 6% 到 7% 的性能提升。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2018-11-06-ruby-2-6-0-preview3-released.md]: x => 倍

**Comment by hlcfan** [comment]
Thanks @dsh0416!

**Comment by dsh0416** [comment]
@hlcfan Thank you so much for your reviewing. The problems mentioned have been fixed.

**Comment by hlcfan** [comment]
👍

**Pull request merged**